test/e2e/app-dir/app/app/(rootonly)/dashboard/hello/page.js

```diff
@@ -8,3 +8,4 @@ export default function HelloPage(props) {
 
 export const runtime = 'experimental-edge'
 export const preferredRegion = ['iad1', 'sfo1']
+export const maxDuration = 10
```

Comment:
consider using another e2e test like app-routes to group these features better? as app is quite full of basic app-router features